src/conversions.jl

```diff
@@ -688,43 +691,21 @@ end
 to_colormap(x::Union{String, Symbol}, n::Integer) = convert_attribute(x, key"colormap"(), n)
 
 """
-A Symbol/String naming the gradient. For more on what names are available please see: `available_gradients()
+A Symbol/String naming the gradient. For more on what names are available please see: `available_gradients()`.
+For now, we support gradients from `PlotUtils` and `ColorBrewer` natively.
```

Comment:
ColorBrewer gradients are also in PlotUtils. No reason to keep them here too.
Also, I think it would be better to list the color libraries that are available, as people will know those (and because of licensing)

Reply:
PlotUtils doesn't have all ColorBrewer gradients unfortunately...if you try `AbstractPlotting.colorbrewer_names in AbstractPlotting.plotutils_names` it returns false.